### PR TITLE
Remove hardcoded

### PR DESCRIPTION
**Pull request by XThomasBU**
placed project specific info to modules/config/project_config.yml

### CODE REVIEW

File: code/modules/dataloader/data_loader.py [hunk 1]
```diff
@@ -434,13 +438,17 @@ def get_chunks(self, uploaded_files, weblinks):
...
-        )
+    # Just for testing
+    (
+        document_chunks,
+        document_names,
+        documents,
+        document_metadata,
+    ) = data_loader.get_chunks(
+        [
+            "https://dl4ds.github.io/fa2024/static_files/discussion_slides/00_discussion.pdf"
```
Comment (trgardos) [inline code/modules/dataloader/data_loader.py:449]: Still hard coded path?

**Review by trgardos — CHANGES_REQUESTED** [review]
I see one hard coded path left. Can that be removed?

File: code/modules/dataloader/data_loader.py (hunk 1 above)
Comment (XThomasBU) [reply code/modules/dataloader/data_loader.py:449]: That is just to test the module, it is not used in the pipeline

Comment (XThomasBU) [reply code/modules/dataloader/data_loader.py:449]: I can add that as an arg

Comment (XThomasBU) [reply code/modules/dataloader/data_loader.py]: Fixed @trgardos

Comment (trgardos) [reply code/modules/dataloader/data_loader.py]: Oops! Format issue.

**Review by trgardos — APPROVED** [review]

**Pull request merged**